lib/web_ui/dev/utils.dart

```diff
@@ -62,6 +62,25 @@ Future<int> runProcess(
   return exitCode;
 }
 
+/// Runs [executable]. Do not follow the exit code or the output.
+void runProcessInTheBackground(
+  String executable,
+  List<String> arguments, {
+  String workingDirectory,
+  bool mustSucceed: false,
+}) async {
+  arguments.add('&');
```

Comment:
Consider using https://api.flutter.dev/flutter/dart-io/ProcessStartMode/detached-constant.html instead.

Another concern is that this will cause `felt` to leak processes. Why do we need to have it to run in the background?

Reply:
It is not the arguments.add('&'); which doesn't do much. (I removed it now)

the problem is, if I use the existing runProcess, which will wait for the exit code.

Reply:
Right, but you are not using `runProcess`. You are using `start`, which does not wait for exit code. My worry is that we won't clean up the driver process, potentially driving the computer out of memory or some other corrupted state.

Reply:
I think we are having a miscommunication :)

I added detached cause I think you wanted it.

For me all I need is adding another method to utils.dart cause I can't use the runProcess method. As you also said, and as I also said, it waits for exit code. We should keep using my method :) 

Please let me know as an actionable item what change do you want?

I also conclude your idea about detached so I'm removing it.

Reply:
Changed the method to the following! Let me know how it looks:
```
/// Runs [executable]. Do not follow the exit code or the output.
void startProcess(
  String executable,
  List<String> arguments, {
  String workingDirectory,
  bool mustSucceed: false,
}) async {
  io.Process.start(
    executable,
    arguments,
    workingDirectory: workingDirectory,
    // Running the process in a system shell for Windows. Otherwise
    // the process is not able to get Dart from path.
    runInShell: io.Platform.isWindows,
    mode: io.ProcessStartMode.inheritStdio,
  );
}
```

Reply:
Yes, the latest version LGTM. Sorry, if there was miscommunication.

Reply:
I added the processes started by this method to common list.

I also added a cleanup in the final step of felt.dart which kills all the processes still running. Does that address your concern?

Reply:
I'll check the code but what you describe makes sense. It's the exact same strategy devicelab uses and it seems to work quite well.

Reply:
Yep, LGTM!